crowdfunding_compassion/models/account_invoice_line.py

```diff
@@ -10,7 +10,4 @@ class AccountInvoiceLine(models.Model):
     crowdfunding_participant_id = fields.Many2one(
         "crowdfunding.participant", "Crowdfunding participant"
     )
-    crowdfunding_project_id = fields.Many2one(
```

Comment:
Could you also add a `onchange` method in the invoice line so that when someone changes the participant from the form view of an invoice line, it automatically sets the other fields `source_id`, `medium_id` and `campaign_id` ?

Reply:
Yes I added this, except for medium_id which is always crowdfunding